docs/getting-started-step-by-step/01-setup.md

```diff
@@ -0,0 +1,92 @@
+# Setup: Setting up Crossplane on Kubernetes.
+
+The goal in this part is to install the Crossplane Runtime and necessary tooling to get started:
+
+* A) Create a local Kubernetes cluster 
+* B) Install the Crossplane CLI
+* C) Install Crossplane Runtime
+
+<details><summary>Prerequisites</summary>
+
+You should bring basic Kubernetes and cloud provider knwowlege as we will use Crossplane to connect both. Also please make sure you have [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [Helm](https://helm.sh/docs/intro/install/) installed.
+</details>
+
+## A) Create a local Kubernetes cluster 
+
+Our first step is to create a local Kubernetes cluster with kind:
```

Comment:
This sounds as if Crossplane should be running in a local k8s cluster. Maybe specify that is only for the examples and that users can accomplish the same goals with the same commands in any other k8s cluster.

Reply:
I agree. I will think about the wording. But a goal of the getting started guide was to make one opinionated path and not provide too many different options as they tend to confuse users.

Reply:
I tried it out in minikube, k3d, kind, EKS, AKS, and GKE. In all those, it works without any changes to the commands so it's probably safe to say "use any Kubernetes cluster you'd like and, if you do not have one already ready, feel free to execute the commands that follow to create a local cluster using the command that follows."

Also, if KinD is our local k8s of choice, make sure to add a link to KinD itself (maybe the installation page). KinD is awesome (the second-best local k8s cluster), but it's still not commonly used. If the idea is to assume what everyone has (without providing a link), that would be minikube or Docker Desktop.

What I'm trying to say is that we should keep it short and quick, but not assume any "special" knowledge.

Reply:
Fixed with https://github.com/luebken/crossplane/pull/1/commits/1fb28783888b6fa044e726924f63d31d023f18d8

Reply:
The one limitation I'm aware of here is around GKE AutoPilot and the permissions it has setup by default. I believe @stevendborrelli has some context here.